package.json

```diff
@@ -141,7 +141,7 @@
     "react-native-plaid-link-sdk": "^10.0.0",
     "react-native-qrcode-svg": "^6.2.0",
     "react-native-quick-sqlite": "^8.0.0-beta.2",
-    "react-native-reanimated": "3.4.0",
+    "react-native-reanimated": "^3.5.4",
```

Comment:
Any reason for adding `^`?

Reply:
You are right when I think about it, it should't be here probably. Gonna remove it 🫡